badges/badges.css

```diff
@@ -0,0 +1,50 @@
+@import url("https://pro.fontawesome.com/releases/v5.10.0/css/all.css");
+
+body {
+  font-family: var(--nunito);
+}
+
+.icon {
+  height: 2rem;
+}
+
+.badge {
+  display: inline-block;
+  position: relative;
+  bottom: 1.3rem;
+  right: 1rem;
+  border-radius: 100%;
+  background-color: var(--red);
+  color: rgb(233, 224, 224);
+  height: 0.9rem;
```

Comment:
using hex is preferable and all colors and common properties should be moved to varaible.

Reply:
done!